src/components/Categoryfilter.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+const CategoryFilter = ({ options, handleChange }) => (
+  <div>
+    <p> Select Category </p>
+    <Select options={options} onChange={(e) => handleChange(e)} required />
```

Comment:
- [ ] The "All" option for the category list should also appear, please add it to comply with the project requirements.

Reply:
Hello @e71az 

'All' option do appear on top of  category list . Kindly have a look at the booklist.js file, I added "all" as an option inside useEffect method.
![Screenshot from 2021-11-03 00-27-52](https://user-images.githubusercontent.com/60758407/139960597-cd7332b6-6d27-45cb-ae3a-39ed03d762b3.png)

Reply:
Sorry, I just noticed the second bar to assign a category to a book but never saw the first one, good luck in your next milestone tanzila!